docs/smart-agent-quickstart.md

```diff
@@ -1,132 +1,311 @@
-# Smart Agent Quick Start
+# SignalFx Smart Agent Next Steps
```

Comment:
I have no control over the name of the file, and I believe there are links to the current file that would break if I change the name. I can change the title of the page, and I have.

Reply:
Would you like to change the title of the page? Suggestions?

Reply:
@keitwb The proposed changes here involve putting the traditional "Quickstart" information into README.md for the repository. With that change I recommend changing the name of this file and perhaps adding a replacement smart-agent-quickstart.md which simply directs users to README.md.

Reply:
Part of the reason for this work is to avoid having people jump from one file to another as much as possible, so keeping a quickstart doc that just tells them to go somewhere else sin't the best solution. 

We need to look again at the docs TOC. Right now the readme contents are ported to a file called Overview. Probably the thing to do is rename that doc to quick-start instead of Overview. Then we can rename this file to next-steps or something like that. 

As for breaking links if we rename this file, I think those could be updated manually (if there are any).

Reply:
Ok, we changed the name of this file to advanced-config-options and the title to Advanced Configuration Options. And I updated the index in PR#554. Overview text is the README and also the contents of the Integrations tile. The advanced-config-options file contains the previous README information. Now the simplified installation instructions will be the first thing new users see. Please approve.